README.md

```diff
@@ -1,78 +1,73 @@
 # ros_gz_project_template
-A template project integrating ROS and Gazebo simulator
+A template project integrating ROS 2 and Gazebo simulator.
 
 ## Included packages
 
-* `ros_gz_example_description` - holds the sdf description of the simulated system and any other assets
+* `ros_gz_example_description` - holds the sdf description of the simulated system and any other assets.
 
-* `ros_gz_example_gazebo` - holds gazebo specific code and configurations.  Namely this is where systems end up.
+* `ros_gz_example_gazebo` - holds gazebo specific code and configurations. Namely this is where systems end up.
 
-* `ros_gz_example_application` - holds ros2 specific code and configurations
+* `ros_gz_example_application` - holds ros2 specific code and configurations.
 
-* `ros_gz_example_bringup` - holds launch files and high level utilities
+* `ros_gz_example_bringup` - holds launch files and high level utilities.
 
 
 ## Install
-### Use as template
-Directly `Use this template` and create your project repository on github.
-
-### On Host System
-##### Requirements
-On Ubuntu Jammy
+### Requirements
 
-1. Install [ROS 2 Humble](https://docs.ros.org/en/humble/index.html)
+1. Choose a ROS and Gazebo combination  https://gazebosim.org/docs/harmonic/ros_installation
+   Note: Ig you're using a specific and unsupported Gazebo version with ROS 2, you might need to set the `GZ_VERSION` environment variable, for example:
 
-1. Install [Gazebo Garden](https://gazebosim.org/docs/garden)
+    ```bash
+    export GZ_VERSION=garden
+    ```
 
 1. Install necessary tools
 
-    `sudo apt install python3-vcstool python3-colcon-common-extensions git wget`
-
-##### Usage
-
-1. Create a workspace, for example:
-
-    ```
-    mkdir -p ~/template_ws/src
-    cd ~/template_ws/src
+    ```bash
+    sudo apt install python3-vcstool python3-colcon-common-extensions git wget
     ```
 
-1. Clone the template:
+### Use as template
+Directly `Use this template` and create your project repository on Github.
```

Comment:
Is this intended to be a link?

Reply:
No this is a tab to clone this repository